src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -131,7 +131,7 @@ case class Command(parts: Seq[CommandPart]) {
     val trimmed = stripAll(s, "\r\n", "\r\n \t")
     val parts = trimmed.split("[\\r\\n]+")
     val indent = parts.map(leadingWhitespaceCount).min
-    parts.map(_.substring(indent)).mkString("")
+    parts.map(_.substring(indent)).mkString("\n")
```

Comment:
Yeah thinking about it more, I can't think of any language that would behave differently if you replaced `[\r\n]+` with `\n` so this may be far more edge-case free than I thought.  This also has the added advantage of compacting the code, which may turn out to be a nice side effect.